### PR TITLE
Fix found typos in amp-html-format.md

### PR DESCRIPTION
**Pull request by johnspurlock**
Fix a few typos found in the spec.

### CODE REVIEW

File: spec/amp-html-format.md [hunk 1]
```diff
@@ -192,8 +192,8 @@ Pseudo selectors are generally forbidden and must be whitelisted 1 by 1. Initial
 
 CSS variables are used to style custom elements.
 
-Class names in author stylesheets may not starts with the string `-amp-`. These are reserved for internal use by the AMP runtime. It follows,
-that user stylesheet may not reference CSS selectors for `-amp-` classes and `i-amp` tags. These classes and elements are not meant to
```
Comment (jmadler) [inline spec/amp-html-format.md]: Well, technically, there's only one user stylesheet.  How about "the user's stylesheet"

**Comment by jmadler** [comment]
Looks good to me, save for the one comment - can you implement and squash?

File: spec/amp-html-format.md (hunk 1 above)
Comment (johnspurlock) [inline spec/amp-html-format.md]: Done.

**Comment by jmadler** [comment]
Thank you so much!

**Comment by johnspurlock** [comment]
np, thanks for the quick merge

**Pull request merged**